### PR TITLE
[MM-676]: Fixed issue of meeting link being posted when user successfully connects to MS Teams Meetings

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
### Summary
Fixed the issue of the meeting link being posted when a user successfully connects to MS Teams Meetings

#### Existing
image
#### Fixed
image
#### Issue link
 Fixes https://github.com/mattermost-community/mattermost-plugin-msteams-meetings/issues/117

### CODE REVIEW

**Review by raghavaggarwal2308 — APPROVED** [review]

File: server/http.go [hunk 1]
```diff
@@ -192,6 +178,14 @@ func (p *Plugin) completeUserOAuth(w http.ResponseWriter, r *http.Request) {
 </html>
 `
 
+	post := &model.Post{
+		UserId:    p.botUserID,
+		ChannelId: channelID,
+		Message:   "User successfully connected to MS Teams Meetings.",
```
Comment (wiggin77) [inline server/http.go:184]: This is just being send to the user, so referring to them as "user" doesn't look right to me.  Perhaps "You have successfully connected to MS Teams Meetings."

Comment (Kshitij-Katiyar) [reply server/http.go]: @wiggin77 
Updated the message to `You have successfully connected to MS Teams Meetings.`
Please have a look. Currently, the disconnect message is `User disconnected from MS Teams Meetings.`
Do you think we should update it too?

**Comment by AayushChaudhary0001** [comment]
@Kshitij-Katiyar I have tested this PR and noticed an inconsistency here, the changes are working fine for the connect slash command(makes sense to what the functionality the slash command should hold), but if we see at the app-bar or channel header, it shows"Start a meeting". And if the user is not connected, this shows a connection link and the user can only connect through it, he need to click again on that button present in app bar/channel header to start a meeting which does not matches with the tooltip.

**Steps to reproduce:-**
1. Disconnect the user's Microsoft account.
2. Click on the Teams meeting icon in app bar/channel header.
3. Connect the account through the link provided.

The user is able to connect the account but need to click again on the icon to start a meeting which does not matches to what it has been shown in the tooltip.
The previous logic was right for the icon in app bar/channel header.
cc: @raghavaggarwal2308

**Comment by raghavaggarwal2308** [comment]
@AayushChaudhary0001 The above issue is fixed now. Please re-test

**Review by AayushChaudhary0001 — APPROVED** [review]
Tested this PR, and the issue reported above has been fixed and tested. Working fine and rest all the cases were found working well as expected, LGTM. Approved.

**Comment by raghavaggarwal2308** [comment]
@wiggin77 Can you please give this PR a review?

**Comment by raghavaggarwal2308** [comment]
@wiggin77 Gentle reminder to review this PR

**Review by wiggin77 — APPROVED** [review]

**Pull request merged**